src/app/pages/ImportAccount.tsx

```diff
@@ -0,0 +1,195 @@
+import * as React from "react";
+import classNames from "clsx";
+import { useForm } from "react-hook-form";
+import { validateMnemonic } from "bip39";
+import {
+  PASSWORD_PATTERN,
+  EMAIL_PATTERN,
+  PASSWORD_ERROR_CAPTION,
+  MNEMONIC_ERROR_CAPTION
+} from "app/defaults";
+
+import Alert from "app/atoms/Alert";
+import FormField from "app/atoms/FormField";
+import FormSubmitButton from "app/atoms/FormSubmitButton";
+
+import PageLayout from "app/layouts/PageLayout";
+import { useThanosFront } from "lib/thanos/front";
+import { navigate } from "lib/woozie";
+
+type TABS = "privateKey" | "fundraiser";
+
```

Comment:
Same, as above :*
Remove line breaks in the imports section, reorder them.

Your import declarations look very cure & beautiful, really!
But, I have already started the project with the rules described above and I want to stick them.